### DISCUSSION TITLE
how to build from source

### DISCUSSION THREAD

**Discussion by 18392152392** [coder/code-server#5779] [Q&A] [answered]
Hello, I was much thirsty for help,  I have download the code-server source code, I want  to build a code-server package, But I don't know how to do?  I have run yarn build and yarn run build:vscode on windows, But it doesn't work,  Can you help me? 

Thank you a lot!

**Comment by lg-nineoneninethreenine** [comment, marked as answer]
There should be a bit more steps to it. Check [CONTRIBUTING.md](https://github.com/coder/code-server/blob/main/docs/CONTRIBUTING.md).

I hope I satiated your thirst for help.

**Comment by WolfLink115** [comment]
Okay, I know those messages earlier were spam messages, but I am not even a part of this code so why was I mentioned by those bots?

**Reply by lg-nineoneninethreenine** [reply]
It seems to be a large-scale attack. Even we don't know how this happened.

**Comment by dotnwat** [comment]
Unrelated: I keep getting pinged on this issue as Github is claiming that I am mentioned in this issue. I don't see a typo mention in any of the previous edits. Github bug?

**Reply by WolfLink115** [reply]
No, there were some bots that pinged some people. I was one of them. Those messages seemed like they have been deleted so I think it is fine.
